examples/federated-kaplan-meier-fitter/run.sh

```diff
@@ -0,0 +1,13 @@
+echo "Starting server"
```

Comment:
```suggestion
#!/bin/bash
set -e
cd "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"/

echo "Starting server"
```

Otherwise the script didn't stop correctly for me.